app/javascript/src/BackendApis/components/IndexPage.js

```diff
@@ -115,7 +115,12 @@ const BackendsIndexPage = ({ backendsCount, backends }: Props) => {
           <Title headingLevel="h1" size="2xl">Backends</Title>
         </LevelItem>
         <LevelItem>
-          <Button variant="primary" component="a" href="/p/admin/backend_apis/new">
+          <Button
+            data-testid="productsIndexCreateBackend-buttonLink"
```

Comment:
@damianpm considering this is the Backends Index page, I think the `id` should be
`backendsIndexCreateBackend-buttonLink`